### DISCUSSION TITLE
Features Suggestion

### DISCUSSION THREAD

**Discussion by magedfathi** [Blair2004/NexoPOS#430] [Ideas]
I would liek to suggest the following points:

1- option to export all reports to excel file for making more analysis from finance team

![export to excel option for all reports](https://user-images.githubusercontent.com/87772530/147558496-d55a7427-c33b-4af8-9b49-60404348120a.PNG)

2- changing the whole sale price to non mandatory filed while adding new product espically restaurant is not using the wholesale price

3- option to Filter the Sale Report, Sales Progress, Sold Stock, Cash Flow, annual report by Sales Person (user) espically if we have multi cashiers (Order Takers " Waiters")

4- option to filter by process status as well 

![Filter By Proces Status](https://user-images.githubusercontent.com/87772530/147558544-81f7137a-9d08-4366-90f8-0bd95020ed7e.PNG)

NexoPos V4.6.8
Platform: Laragon on Windows 10 64 bit

**Comment by Blair2004** [comment]
Hi, all approved.
Thank you @magedfathi
